parser/src/main/java/io/specmesh/apiparser/model/ApiSpec.java

```diff
@@ -22,16 +22,19 @@
 import java.util.LinkedHashMap;
 import java.util.Map;
 import java.util.stream.Collectors;
-import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.Value;
 import lombok.experimental.Accessors;
 
 /** Pojo representing the api spec */
-@Value
+@Builder
+@Data
 @Accessors(fluent = true)
 @JsonIgnoreProperties(ignoreUnknown = true)
-@NoArgsConstructor(force = true, access = AccessLevel.PRIVATE)
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Don't think these are needed for this PR. Not having them reduces the size of your API, which is a good thing.

Also, consider if this API lets the user build specs that are invalid.  This should be avoided. (I think lombok supports _required_ properties, which you should set)

Reply:
its needed for the lombok @Builder

Reply:
don't think it is. Seems to compile just fine without..

Reply:
now using
@AllArgsConstructor(access = AccessLevel.PRIVATE)
@NoArgsConstructor(access = AccessLevel.PRIVATE)